File: Day-3/exercise-5.py
# 🚨 Don't change the code below 👇
print("Welcome to the Love Calculator!")
name1 = input("What is your name? \n")
name2 = input("What is their name? \n")
# 🚨 Don't change the code above 👆

#Write your code below this line 👇
name1 = name1.lower()
name2 = name2.lower()
name = name1 + name2
true = name.count('t') + name.count('r') + name.count('u') + name.count('e')
love = name.count('l') + name.count('o') + name.count('v') + name.count('e')

love_score = true*10 + love

if(love_score < 10 or love_score > 90):
    print(f"Your score is {love_score}, you go together like coke and mentos.")
elif(love_score >= 40 and love_score <= 50):
    print(f"Your score is {love_score}, you are alright together.")
else:
    print(f"Your score is {love_score}.")

File: Day-3/exercise-5.cpp
#include <iostream>
#include <string>
#include <algorithm>

int main(){
    std::string name1{}, name2{};
    int true_count{}, love_count{}, score{};
    std::cout << "What is your name? "; std::getline(std::cin, name1);
    std::cout << "What is their name? "; std::getline(std::cin, name2);

    name1 = name1.append(name2);
    std::for_each(name1.begin(), name1.end(), [](char &c){c = std::tolower(c);});

    for(const char &i : name1){
        if(i == 't' || i == 'r' || i == 'u'){
            true_count++;
        }else if(i == 'l' || i == 'o' || i == 'v'){
            love_count++;
        }else if(i == 'e'){
            true_count++;
            love_count++;
        }
    }
    score = true_count*10 + love_count;

    if(score < 10 || score > 90){
        std::cout << "Your score is " << score << ", you go like coke and mentos." << std::endl;
    }else if(score >= 40 && score <= 50){
        std::cout << "Your score is " << score << ", you go alright together." << std::endl;
    }else{
        std::cout << "Your score is " << score << "." << std::endl;
    }

    return 0;
}